src/config.ts

```diff
@@ -64,11 +64,14 @@ export interface Config extends AuthenticationConfig {
 
 // Default values for configuration for a profiler.
 export const defaultConfig: Config = {
-  projectId: '{{projectId}}',
   logLevel: 1,
+  projectId: '{{projectId}}',
   serviceContext: {
+    service: 'default',
```

Comment:
Like in Go agent, we should require setting the service name, it should not fall back to anything. Except on Flex where it can be derived from the environment so the user is not required to set it (but if they do, that will override the environment setting).